src/Symfony/Component/TypeInfo/CHANGELOG.md

```diff
@@ -5,7 +5,7 @@ CHANGELOG
 ---
 
  * Add `Type::accepts()` method
- * Add `TypeFactoryTrait::fromValue()` method
+ * Add the `TypeFactoryTrait::fromValue()`, `TypeFactoryTrait::arrayShape()`, and `TypeFactoryTrait::arrayKey()` methods
```

Comment:
Maybe we can merge the `accepts` method as well?

Reply:
`accepts()` was documented for `Type` as it's not in the trait. Does that really make sense to merge them or isn't it more readable this way?

Reply:
These methods are most likely to be accessed via the `Type` class. But we can keep them separate in the changelog.